package.json

```diff
@@ -25,7 +25,8 @@
   "eslintConfig": {
     "extends": [
       "react-app",
-      "react-app/jest"
+      "react-app/jest",
+      "eslint:recommended"
```

Comment:
I'm thinking of removing estlint because that was the biggest consistent complaint for everyone - I think we had originally put in a really strong and strict estlint rule here though, thoughts?

Reply:
> I'm thinking of removing estlint

@bonnieli Personally, I think removing eslint is okay. However, some candidates might find it easier to work **with eslint** because it catches mistakes like unused variables.

> Environment key "jest/globals" is unknown

I experienced a issue similar to this. The cause was duplicate dependencies in `devDependencies` (e.g. `eslint`, `eslint-config-airbnb`). I fixed the dependency tree, so try a clean install (`rm -rf node_modules && yarn`) to see if the issue persists.

Reply:
Can you remove estlint? We definitely get more complaints and frustration than it helps them catch unused variables :P

Reply:
and doing the clean install worked! thanks :)

Reply:
wait, I thought we are removing this one line (eslint:recommended)